frontend/packages/dev-console/src/components/monitoring/overview/queries.ts

```diff
@@ -0,0 +1,78 @@
+import { template } from 'lodash';
+import {
+  humanizeBinaryBytes,
+  humanizeCpuCores,
+  humanizeDecimalBytesPerSec,
+} from '@console/internal/components/utils';
+import { ByteDataTypes } from '@console/shared/src/graph-helper/data-utils';
+import { GraphTypes } from '../dashboard/MonitoringDashboardGraph';
+
+export const workloadMetricQueries = [
```

Comment:
/cc @openshift/openshift-team-monitoring

Reply:
these seem pretty duplicate to the grafana dashboards we're embedding into the console, do we really want multiple pages that show the same data?

Reply:
@brancz considering feature freeze in 2 days we decided to continue with pre-cannef queries for now. Post that will take a spike to see how we can leverage dashboard work done for monitoring using configmaps for Grafna  cc @abhi-kn 

https://coreos.slack.com/archives/CR8NP36LT/p1579709940016100?thread_ts=1579709940.016100&cid=CR8NP36LT